src/App.tsx

```diff
@@ -1,61 +1,134 @@
+import { useState } from 'react';
 import './App.scss';
+import { TodoList, TodoType, UserType } from './components/TodoList';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+interface Errors {
+  title: string;
+  user: string;
+}
 
 export const App = () => {
+  const [title, setTitle] = useState<string>('');
+  const [author, setAuthor] = useState<string>('0');
```

Comment:
```suggestion
  const [title, setTitle] = useState('');
  const [author, setAuthor] = useState('0');
```
it's redundant to write types for primitives